app/assess/routes.py

```diff
@@ -87,6 +90,7 @@ def application_sub_crit_scoring(application_id: str, sub_criteria_id: str):
         "sub_criteria.html",
         scores_submitted=scores_submitted,
         form=form,
+        score_list=score_list if len(score_list) > 0 else None,
```

Comment:
Would this be better defaulted to an empty collection?
```suggestion
        score_list=score_list if len(score_list) > 0 else [],
```